apps/home/templates/index.html

```diff
@@ -19,9 +19,11 @@ <h1 class="display-3">{{settings.TAG_LINE_1}}</h1>
   <p>{{settings.EXPLAINATION_LINE}}</p>
   <p class="lead">
  
- 
+   {% if settings.REQUIRE_INVITE_TO_REGISTER %}
```

Comment:
This button should be removed completely, as we want to manually create an account via offline request